packages/dx-grid-core/src/plugins/table-column-visibility/helpers.ts

```diff
@@ -0,0 +1,6 @@
+import { TABLE_DATA_TYPE } from '../table/constants';
+import { TableColumn } from '../../types';
+
+export const tableDataColumnsExist = (tableColumns: TableColumn[]) => tableColumns.some(
```

Comment:
Why is return type (`boolean`) not defined?

Reply:
It is inferred from usage: TS knows that Array.some returns boolean

Reply:
OK.